### PR TITLE
Make terminal inline chat look more like editor inline chat

### PR DESCRIPTION
**Pull request by Tyriar**
Big changes:

- No more border radius, it looks like a monaco zone widget now.
- Adjusted paddings to make it look right
- Added border animation like editor inline chat
- Removed close chat button

Fixes #233199

(gif also includes https://github.com/microsoft/vscode/pull/233210 to improve code block alignment)

Recording 2024-11-06 at 07 22 15

### CODE REVIEW

File: src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts [hunk 1]
```diff
@@ -44,7 +45,7 @@ const enum Message {
 	ReturnInput = 1 << 6,
 }
 
-const terminalChatPlaceholder = localize('default.placeholder', "Ask how to do something in the terminal");
+const terminalChatPlaceholder = localize('default.placeholder', "Ask Copilot");
```
Comment (jrieken) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts:48]: We shouldn't hardcode product names here. You can use the chat service to lookup the default agent for your location and use its name.

File: src/vs/workbench/contrib/terminalContrib/chat/browser/media/terminalChatWidget.css [hunk 2]
```diff
@@ -3,27 +3,71 @@
...
 }
 
-.terminal-inline-chat .inline-chat {
+.monaco-workbench .terminal-inline-chat .inline-chat {
 	margin-top: 0 !important;
+	color: inherit;
+	border-radius: unset;
+	border: unset;
+	box-shadow: unset;
+	background: var(--vscode-inlineChat-background);
```
Comment (jrieken) [inline src/vs/workbench/contrib/terminalContrib/chat/browser/media/terminalChatWidget.css:20]: We might be able to do away with the unsettling  and simply remove some styles here https://github.com/microsoft/vscode/blob/f6686e3ac767891e438e906b9b83154ad880e0c6/src/vs/workbench/contrib/inlineChat/browser/media/inlineChat.css#L7

Comment (Tyriar) [reply src/vs/workbench/contrib/terminalContrib/chat/browser/media/terminalChatWidget.css:20]: I agree. I'd prefer to leave this to you though as I'm not that familiar with all the places inline chat is used now and would hate to break other UI.

**Comment by Tyriar** [comment]
Welcome message: https://github.com/microsoft/vscode-copilot/pull/10254

**Review by jrieken — APPROVED** [review]

**Pull request merged**